### PR TITLE
Commit 12 - changes for tooltips for role manager table

### PR DESCRIPTION
**Pull request by Trjegul84**
Changes related to commit 12 of Ceiba

### CODE REVIEW

File: src/main/webapp/WEB-INF/pages/manage/home.ftl [hunk 1]
```diff
@@ -6,14 +6,10 @@
 <#include "/WEB-INF/pages/macros/forms.ftl"/>
 <#include "/WEB-INF/pages/macros/popover.ftl">
 <#include "/WEB-INF/pages/macros/resourcesTable.ftl"/>
-<script type="text/javascript" src="${baseURL}/js/jquery/jquery-3.5.1.min.js"></script>
```
Comment (alegnaaived) [inline src/main/webapp/WEB-INF/pages/manage/home.ftl:9]: Why did you remove this line?

Comment (alegnaaived) [inline src/main/webapp/WEB-INF/pages/manage/home.ftl:9]: This line was removed to the library jquery.js, and it is a different version. But, looks like the right change.

**Review by alegnaaived — APPROVED** [review]

File: src/main/webapp/WEB-INF/pages/manage/home.ftl (hunk 1 above)
Comment (Trjegul84) [reply src/main/webapp/WEB-INF/pages/manage/home.ftl:9]: I dont know to this is affecting. We have to take in mind this.
